cronjob/templates/cronjob.yaml

```diff
@@ -10,6 +10,8 @@ kind: CronJob
 metadata:
   name: {{ .Values.name }}
   namespace: {{ .Release.Namespace }}
+  labels:
+    businessid: {{ required "businessid must be provided" .Values.businessid | quote }}
```

Comment:
NIT: why add required here, should add required in all businessid field?

Reply:
This is the entry point, so I believe checking here would be enough.